src/data/repositories/UserD2ApiRepository.ts

```diff
@@ -231,19 +280,20 @@ export class UserD2ApiRepository implements UserRepository {
         const userIds = users.map(user => user.id);
 
         return this.getFullUsers({ filters: { id: ["in", userIds] } }).flatMap(existingUsers => {
-            return this.getGroupsToSave(users, existingUsers).flatMap(userGroups => {
-                const usersToSend = _(existingUsers)
-                    .map(existingUser => {
-                        const user = users.find(user => user.id === existingUser.id);
-                        if (!user) return undefined;
-                        return this.buildUsersToSave(existingUser, user);
-                    })
-                    .compact()
-                    .value();
-
-                return apiToFuture(this.api.metadata.post({ users: usersToSend, userGroups })).flatMap(data => {
-                    return this.saveLocales(usersToSave).map(() => data);
-                });
+            const usersToSend = _(existingUsers)
+                .map(existingUser => {
+                    const user = users.find(user => user.id === existingUser.id);
+                    if (!user) return undefined;
+                    return this.buildUsersToSave(existingUser, user);
+                })
+                .compact()
+                .value();
+
+            return apiToFuture(this.api.metadata.post({ users: usersToSend })).flatMap(data => {
+                return Future.joinObj({
+                    saveLocales: this.saveLocales(usersToSave).map(() => data),
```

Comment:
I was that `.map(() => data)` was a left-over when refactoring, it's not needed here.